library/src/main/java/com/venmo/android/pin/PinSdkFragment.java

```diff
@@ -2,49 +2,41 @@
 
 import android.app.Activity;
 import android.app.Fragment;
+import android.content.Context;
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
 
-public class PinFragment extends Fragment {
+public class PinSdkFragment extends Fragment implements PinFragmentImplement {
 
     private static final String KEY_FRAGMENT_VIEW_TYPE = "com.venmo.input_fragment_view_type";
 
-    private Listener mListener;
+    private PinListener mListener;
     private PinDisplayType mPinDisplayType;
     private BaseViewController mViewController;
     private PinFragmentConfiguration mConfig;
     private View mRootView;
 
-    public enum PinDisplayType {
-        VERIFY, CREATE, CONFIRM
-    }
-
-    public interface Listener {
-        public void onValidated();
-        public void onPinCreated();
-    }
-
-    public static PinFragment newInstanceForVerification() {
+    public static PinSdkFragment newInstanceForVerification() {
         return newInstanceForVerification(null);
     }
 
-    public static PinFragment newInstanceForVerification(PinFragmentConfiguration config) {
+    public static PinSdkFragment newInstanceForVerification(PinFragmentConfiguration config) {
         return newInstance(PinDisplayType.VERIFY, config);
     }
 
-    public static PinFragment newInstanceForCreation() {
+    public static PinSdkFragment newInstanceForCreation() {
         return newInstanceForCreation(null);
     }
 
-    public static PinFragment newInstanceForCreation(PinFragmentConfiguration config) {
+    public static PinSdkFragment newInstanceForCreation(PinFragmentConfiguration config) {
         return newInstance(PinDisplayType.CREATE, config);
     }
 
-    private static PinFragment newInstance(PinDisplayType type, PinFragmentConfiguration config) {
-        PinFragment instance = new PinFragment();
+    private static PinSdkFragment newInstance(PinDisplayType type, PinFragmentConfiguration config) {
```

Comment:
Can we name this as `PinFragment`?
Also add java docs to both fragment classes.